### PR TITLE
Validating Manifest Schema

### LINKED ISSUE #226: Expose function to validate Election Description
Create a function that exposes the `ElectionDescription` essentially the `ElectionManifest` method of `isValid` so an ElectionDescription can be passed in and a boolean is return on whether the description is valid. 

```
def manifest_is_valid(manifest) : boolean
```

### PR DESCRIPTION
**Pull request by keithrfung**
### Issue

Fixes #226 

### Description
Add a method to validate a manifest. 

Major points to review
- What is required
- What is string length limited
- What are the minimums.

### Testing
`test_schema.py`

### Checklist
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] 🤔 **CONSIDER** adding a unit test if your PR resolves an issue.
- [x] ✅ **DO** check open PR's to avoid duplicates.
- [x] ✅ **DO** keep pull requests small so they can be easily reviewed.
- [x] ✅ **DO** build locally before pushing.
- [x] ✅ **DO** make sure tests pass.
- [x] ✅ **DO** make sure any new changes are documented.
- [x] ✅ **DO** make sure not to introduce any compiler warnings.
- [x] ❌**AVOID** breaking the continuous integration build.
- [x] ❌**AVOID** making significant changes to the overall architecture.

💚Thank you!

### CODE REVIEW

File: src/electionguard/election_description_schema.json [hunk 1]
```diff
@@ -0,0 +1,585 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
```
Comment (keithrfung) [inline src/electionguard/election_description_schema.json:2]: This is using `draft-07`

Comment (keithrfung) [reply src/electionguard/election_description_schema.json:2]: https://json-schema.org/specification.html

File: src/electionguard/election_description_schema.json [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+            "properties": {
+                "address_line": {
+                    "title": "Full Address",
+                    "type": [
+                        "array",
+                        "null"
+                    ],
+                    "items": {
+                        "type": "string",
+                        "maxLength": 256
```
Comment (keithrfung) [inline src/electionguard/election_description_schema.json:47]: Currently all the `string` have a limit of 256. We could just provide a default implementation with no limitations.

File: src/electionguard/election_description_schema.json [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+                        "type": [
+                            "string",
+                            "null"
+                        ]
+                    }
+                }
+            }
+        }
+    },
+    "required": [
```
Comment (keithrfung) [inline src/electionguard/election_description_schema.json:580]: The `required` could be at multiple levels to enforce the scheme.

File: src/electionguard/schema.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+def _load_schema(json_schema_file_name: str) -> Any:
+    """Loads the given schema"""
+    with open(join(dirname(realpath(__file__)), json_schema_file_name), "r") as file:
+        schema = load(file)
+    return schema
+
+
+election_description_schema = _load_schema("election_description_schema.json")
```
Comment (rkorsak) [inline src/electionguard/schema.py:17]: I'm not 100% confident of this, but I feel like this shouldn't be run automatically on import like this, i.e. any code that even imports this module will wait for the schema to load during import.

Would putting it behind a function be better?

Comment (keithrfung) [reply src/electionguard/schema.py:17]: I originally had something like this. I thought it made it more convenient to have it like this. I originally had a get_election_description_schema. I can reimplement quickly.

**Review by rkorsak — APPROVED** [review]
✨ :shipit:  ✨

**Pull request merged**